frontend/src/services/apiService.js

```diff
@@ -33,7 +33,8 @@ const fetchJsonContent = (url, authToken) => {
     .then((res) => res.json());
 };
 
-export const fetchConfig = () => fetchJsonContent(preparePath('/config'));
+export const fetchConfig = (isMockEnv = false) =>
+  isMockEnv ? Promise.resolve({}) : fetchJsonContent(preparePath('/config'));
```

Comment:
Why is this changes required?

Reply:
running locally with mock data didn't work without this change, but you are write it is out of scope for this PR, will remove it

Reply:
I am not sure, have you tried the command mentioned on https://github.com/otto-de/gitactionboard/blob/main/CONTRIBUTING.md#run-only-frontend-with-mock-data ?